service/main.ml

```diff
@@ -121,9 +121,8 @@ let internal_workers =
   Arg.value
   @@ Arg.opt Arg.int 20
   @@ Arg.info
-       ~doc:"The number of threads that can handle more requests in parallel"
-       ~docv:"N"
-       [ "internal-thread-workers" ]
+       ~doc:"The number of process that can handle more requests in parallel"
```

Comment:
What about this?
> The number of sub-processes solving requests in parallel.

Reply:
Yes good idea, all the process are spawn using `Lwt_process` which forks the current process.